source/includes/steps-install-verify-files-windows.yaml

```diff
@@ -0,0 +1,129 @@
+
+  .. example::
+
+     To download the SHA256 signature for the ``v3.4-latest`` release 
+     for Windows using Powershell, invoke this command:
+
+     .. code-block:: powershell
+
+        Invoke-WebRequest -Uri "https://downloads.mongodb.org/win32/mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi.sha256" `
+          -OutFile "$Env:HomePath\Downloads\mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi.sha256"
+---
+title: Verify the signature of the MongoDB installer.
+stepnum: 4
+level: 4
+ref: verify-sig
+content: |
+
+  Invoke ``sigcheck``:
+
+  .. code-block:: powershell
+
+     $Env:ProgramFiles\Sigcheck\sigcheck64.exe `
+       -h $Env:HomePath\Downloads\mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi
+
+  .. note::
+     Click :guilabel:`Agree` to accept the 
+     :abbr:`EULA (End User License Agreement)` when it displays. 
+
+  ``sigcheck`` returns this verification information for the latest 
+  release of MongoDB 3.4:
+
+  .. code-block:: bat
+     :emphasize-lines: 19
+
+     Sigcheck v2.60 - File version and signature viewer
+     Copyright (C) 2004-2017 Mark Russinovich
+     Sysinternals - www.sysinternals.com
+
+     $Env:HomePath\downloads\mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi:
+             Verified:       Signed
+             Signing date:   12:34 AM 6/20/2018
+             Publisher:      MongoDB, Inc.
+             Company:        n/a
+             Description:    n/a
+             Product:        n/a
+             Prod version:   n/a
+             File version:   n/a
+             MachineType:    n/a
+             MD5:    D7866C013989AEE2FA87774EFFF884F0
+             SHA1:   E5D7D78E8FFFF9CFF3BD605C3407A55F87F4C8DD
+             PESHA1: E5D7D78E8FFFF9CFF3BD605C3407A55F87F4C8DD
+             PE256:  8FE0670DF2AB74CCD33910C0AF2F000225BA2ED21330767D95E3F6DED96E6893
+             SHA256: 8FE0670DF2AB74CCD33910C0AF2F000225BA2ED21330767D95E3F6DED96E6893
+             IMP:    n/a
+
+---
+title: Compare the Signature File to the MongoDB Installer Hash.
+stepnum: 5
+level: 4
+ref: download-key-file
+content: |
+
+  To compare the signature file to the hash of the MongoDB binary, 
+  invoke this Powershell script:
+
+  .. code-block:: powershell
+
+     $sigHash = (Get-Content $Env:HomePath\Downloads\mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi.sha256 | Out-String).SubString(0,64).ToUpper(); `
+     $fileHash = (Get-FileHash $Env:HomePath\Downloads\mongodb-win32-x86_64-2008plus-ssl-v3.4-latest-signed.msi).Hash.Trim(); `
+     echo $sigHash; echo $fileHash; `
+     $sigHash -eq $fileHash
+
+  .. code-block:: bat
+     :emphasize-lines: 1-2
+
+     8FE0670DF2AB74CCD33910C0AF2F000225BA2ED21330767D95E3F6DED96E6893
+     8FE0670DF2AB74CCD33910C0AF2F000225BA2ED21330767D95E3F6DED96E6893
+     True
+
+  The command outputs three lines:
+  
+  - An SHA256 hash that you downloaded directly from MongoDB.
+  - An SHA256 hash computed from the MongoDB binary you
```

Comment:
Nit: Should this and the line above be "A SHA256 hash..."

Reply:
Depends how you pronounce it? In my head, I'm saying it "ESS-aych-ay" and not "SHAH". I'll change it, though, because it makes sense to do so here.

Reply:
OH, THAT MAKES SENSE NOW! I read it as SHAH... haha